src/openklant/components/klantinteracties/tests/test_validators.py

```diff
@@ -6,56 +6,70 @@
     PartijIdentificatorCodeRegister,
     PartijIdentificatorCodeSoortObjectId,
 )
+from openklant.components.klantinteracties.models.tests.factories.partijen import (
+    PartijIdentificatorFactory,
+)
 from openklant.components.klantinteracties.models.validators import (
-    PartijIdentificatorValidator,
+    PartijIdentificatorTypesValidator,
+    PartijIdentificatorUniquenessValidator,
 )
 
 
-class PartijIdentificatorValidatorTests(TestCase):
+class PartijIdentificatorTypesValidatorTests(TestCase):
```

Comment:
This is missing a testcase for `validate_unique_partij_identificator_locally`, let's add one with `raise NotImplementedError` to make sure it's not forgotten